kubernetes-model-generator/kubernetes-model-admissionregistration/pom.xml

```diff
@@ -32,6 +32,83 @@
+      io.fabric8.kubernetes.api.model.flowcontrol.v1beta1,
+      io.fabric8.kubernetes.api.model.metrics.v1beta1,
+      io.fabric8.kubernetes.api.model.networking.v1,
+      io.fabric8.kubernetes.api.model.networking.v1beta1,
+      io.fabric8.kubernetes.api.model.node.v1alpha1,
+      io.fabric8.kubernetes.api.model.node.v1beta1,
+      io.fabric8.kubernetes.api.model.policy.v1,
+      io.fabric8.kubernetes.api.model.policy.v1beta1,
+      io.fabric8.kubernetes.api.model.rbac,
+      io.fabric8.kubernetes.api.model.scheduling.v1,
+      io.fabric8.kubernetes.api.model.scheduling.v1beta1,
+      io.fabric8.kubernetes.api.model.storage,
+      io.fabric8.kubernetes.api.model.storage.v1beta1,
+      io.fabric8.openshift.api.model;resolution:=optional,
+      io.fabric8.openshift.api.model.clusterautoscaling.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.clusterautoscaling.v1beta1;resolution:=optional,
+      io.fabric8.openshift.api.model.runtime;resolution:=optional,
+      io.fabric8.openshift.api.model.console.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.console.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.monitoring.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.monitoring.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.operator;resolution:=optional,
+      io.fabric8.openshift.api.model.operator.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.operator.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.operator.controlplane.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.operatorhub.manifests;resolution:=optional,
+      io.fabric8.openshift.api.model.operatorhub.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.operatorhub.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.operatorhub.lifecyclemanager.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.machineconfig.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.machine.v1beta1;resolution:=optional,
+      io.fabric8.openshift.api.model.tuned.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.whereabouts.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.storageversionmigrator.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.miscellaneous.cloudcredential.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.miscellaneous.cncf.cni.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.miscellaneous.imageregistry.operator.v1;resolution:=optional,
+      io.fabric8.openshift.api.model.miscellaneous.metal3.v1alpha1;resolution:=optional,
+      io.fabric8.openshift.api.model.miscellaneous.network.operator.v1;resolution:=optional,
+    </osgi.import>
+    <osgi.export>
+      io.fabric8.kubernetes.api,
+      io.fabric8.kubernetes.api.builder,
+      io.fabric8.kubernetes.api.model**,
```

Comment:
This should b same as `<Export-Package>` plugin configuration you deleted below

Reply:
Done